docs/content/open-api/openapi.yaml

```diff
@@ -0,0 +1,1296 @@
+                  enum:
+                    - enter
+                    - exit
+                    - eval
+                    - fail
+                    - redo
+                  example: enter
+                query_id:
+                  type: number
+                  description: The query that the trace event was emitted for. Use this field to distinguish trace events emitted by from different queries.
+                  example: 0
+                  minimum: 0
+                parent_id:
+                  type: number
+                  description: |-
+                    The parent query. Use this field to identify trace events from related queries.
+
+                    For example, if query A references rule R, trace events emitted when evaluating rule R will have the *parent_id* field set to query A’s *query_id*.
+                  example: 0
+                  minimum: 0
+                type:
+                  type: string
+                  description: The type of the **node** field
+                  example: expr
+                  enum:
+                    - expr
+                    - rule
+                    - body
+                node:
+                  type: object
+                  description: 'The AST element associated with the evaluation step.'
+                  externalDocs:
+                    description: AST
+                    url: 'https://godoc.org/github.com/open-policy-agent/opa/ast'
+                  properties:
+                    index:
+                      type: number
+                      description: Node number
+                    terms:
+                      type: array
+                      items:
+                        type: object
+                        properties:
+                          type:
+                            type: string
+                            example: var
+                          value:
+                            type: string
+                            example: $term1
+                locals:
+                  type: array
+                  description: The query's term bindings at the point when the trace event was emitted.
+                  items:
+                    type: object
+                    properties:
+                      key:
+                        type: object
+                        properties:
+                          type:
+                            type: string
+                          value:
+                            type: string
+                      value:
+                        type: object
+                        properties:
+                          type:
+                            type: string
+                          value:
+                            type: string
+  securitySchemes: {}
+security: []
```

Comment:
If possible we might want to add instructions for running OPA with TLS and other hardening techniques here. See https://www.openpolicyagent.org/docs/latest/security/#hardened-configuration-example

Reply:
In terms of the Open API Specification, this is the place you pop in the authentication methods for testing 'try it out' calls (see previous comment).

If you've got this already in your security/authentication docs, I think you are covered.

Reply:
👍